src/components/Article.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom"
+import { getArticle } from "../../api";
+
+
+const Article = () =>{
+const [article, setArticle] = useState()
+const article_id = useParams();
+const [isLoading, setIsLoading] = useState(true)
+const[error, setError] = useState(true)
```

Comment:
We are starting off with an error that is true, meaning that your conditional rendering that uses this state may show up when we don't want it to. In terms of the flow we shouldn't be setting the error state to true, until there is an error.